src/components/ViewerComponent.vue

```diff
@@ -32,11 +32,12 @@
 
 <script>
 import { getSharingToken } from '../helpers/token.js'
+import getEditorInstance from './Editor.singleton.js'
 
 export default {
 	name: 'ViewerComponent',
 	components: {
-		Editor: () => import(/* webpackChunkName: "editor" */'./Editor.vue'),
+		Editor: async () => { return await getEditorInstance() },
```

Comment:
I think this could be simplified:

```suggestion
		Editor: getEditorInstance,
```